pages/content/posts/tpac-sevilla.md

```diff
@@ -0,0 +1,44 @@
+---
+author: "Florian Scholz"
+title: "OWD at TPAC Sevilla"
+date: "2023-09-21"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "Open Web Docs traveled to Seville, Spain to attend the W3C's annual TPAC conference."
+---
+
+Last week I traveled to Seville, Spain to attend the [W3C TPAC 2023 conference](https://www.w3.org/2023/09/TPAC/).
+
+Every year, the W3C <dfn id="tpac"><abbr title="Technical Plenary and Advisory Committee">TPAC</abbr></dfn> conference invites W3C members and boards, W3C Working and Interest Groups, as well as Invited Experts and others to meet and discuss work on web standards and web technologies. This time it was a hybrid event - participation was possible in-person and remotely.
+
+It was my first in-person TPAC event and enjoyed meeting a lot of folks that I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.
+I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs) and I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some even watch our [GitHub project repository](https://github.com/openwebdocs/project) to hear all about our new projects immediately (cheers, Thomas Steiner!).
+
+If you attend an in-person event these days, it really is about the people. So, let me give a shout out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).
+Since our founding, we only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)) and so it was lovely to reconnect and talk about web platform documentation once again.
```

Comment:
```suggestion
Every year, the W3C <dfn id="tpac"><abbr title="Technical Plenary and Advisory Committee">TPAC</abbr></dfn> conference invites W3C members and boards, W3C Working and Interest Groups, as well as Invited Experts and others to meet and discuss work on web standards and web technologies. This time, it was a hybrid event - participation was possible both in-person and remotely.

It was my first in-person TPAC event. I enjoyed meeting a lot of folks that had I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.

I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs). I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some TPAC attendees I spoke with even watch our [GitHub project repository](https://github.com/openwebdocs/project) to immediately hear all about our new projects (cheers, Thomas Steiner!).

Attending an in-person event these days is really about the people. So, let me give a shout-out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).

Since our founding, we have only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)). It was so lovely to reconnect and talk about web platform documentation once again.
```

some grammar suggestions. nothing major. Did you want to link the names to linkedIn or fedi accounts?